create-budget/README.md

```diff
@@ -0,0 +1,19 @@
+# Create Budget to track cost or usage
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F2create-budget%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fcreate-budget%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template shows how to create a budget to track cost/usage and get notified whenever a specified threshold is met.
+
+The accompanying PowerShell script shows how to create a resource group and budget under that resource group from the template. Before running the script, edit *azuredeploy.parameters.json* and replace the values marked with *'#####'* and *'YYYY-MM-DD'* and other sample data.
```

Comment:
update this:
- there's no script
- the parameter updates need to be more detailed given the failures we had

Reply:
Removed the mention about script. Also added detailed information regarding parameters.